google/resource_google_project_test.go

```diff
@@ -216,6 +216,52 @@ func TestAccProject_appEngineBasic(t *testing.T) {
 	})
 }
 
+func TestAccProject_appEngineUpdate(t *testing.T) {
+	t.Parallel()
+
+	org := getTestOrgFromEnv(t)
+	pid := acctest.RandomWithPrefix("tf-test")
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccProject_appEngineNoApp(pid, org),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
If you want to keep the checks I won't tell you no, but they aren't necessary if you check by importing

Reply:
I think in this case import wouldn't actually check them, because they're computed fields not set in the config, so I believe without the checks, if the fields just aren't being set, ImportStateVerify won't complain. Because (from what I understand) ImportStateVerify just creates a new, empty state, imports the resource again, and checks it against the state from creating the resource. If there was a bug where these weren't being set, in both cases it would have them not set, so ImportStateVerify would think that's fine. And because they're computed and not config fields, there'd be no diff.

Reply:
Cool, makes sense! Regardless, you can still take out `testAccCheckGoogleProjectExists`